plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/tool/display/ToolAction.java

```diff
@@ -22,22 +22,31 @@
 public class ToolAction extends Action {
 
     ToolProxy tool;
+    private boolean checked = true;
 
     ToolAction( ToolProxy proxy ) {
         this.tool = proxy;
+        checked = tool.isToggelingButtonEnabled();
```

Comment:
can we move this line into if-block

```
if(tool.isToggleButton()) {
  checked = tool.isToggelingButtonEnabled();
  setChecked(checked);
}
```

and initialize checked with false; as default?

Reply:
yes we can